tests/parsing/ConfParserTest.py

```diff
@@ -200,3 +200,46 @@ def test_settings_override_warning(self):
                                             'already been defined in section '
                                             'name. The previous setting will '
                                             'be overridden.')
+
+
+class ConfParserTestParselinesSections(unittest.TestCase):
+    test_file = """
+    [basic]
+
+    ignore = aaa,bbb
+    ignore += ccc
+    ignore += ddd
+
+    [with_unrelated_settings]
+    ignore = aaa,bbb
+    other = variable
+    ignore += ccc
+    ignore += ddd
+
+    [multiline]
+    ignore = aaa,
+    bbb
+    other = variable
+    ignore = ccc,
+    ddd
+    """
+
+    def setUp(self):
+        self.tempdir = tempfile.gettempdir()
+        self.file = os.path.join(self.tempdir, '.coafile')
+        with open(self.file, 'w') as file:
+            file.write(self.test_file)
+        self.uut = ConfParser()
+        self.sections = self.uut.parse(self.file)
+
+    def tearDown(self):
+        os.remove(self.file)
+
+    def test_multiple_ignore_in_single_settings(self):
+        expected = ['aaa', 'bbb', 'ccc', 'ddd']
+        for section in self.uut.sections:
```

Comment:
I feel like the tests would make much more meaning if the final value of the `ignore` field is different to test against in case of different sections, currently according to the `.coafile` you have simulated for this test, its all `['aaa', 'bbb', 'ccc', 'ddd']` for all. Can you vary the value? Also please respond to @jayvdb 's question on testing it in case of section inheritance, whether it works or not, if it doesn't opening issues about it and referencing them on this issue/PR

Reply:
and if it does then adding tests for that too!

Reply:
also circle shows incompliant PEP8 code. Please fix that.